### ISSUE TITLE
Change 'ts' to utc 

### ISSUE THREAD

**Issue by mquade** [Hofei90/smartmeter#14]
Visualisation in Grafana is much easier if the timestamp is stored in utc and not in the timezone-format. Currently it is stored in the timezone. For me this gives glitches (+ or - the UTC-Offset). I have to work around it by converting the ts field in the query. 
It would be much less pain if you change the timeformat either to utc or let the DB do the work - in mysql it would be possible to use "on update current timestamp" or now(). Then the time stamp is always stored in utc and only converted to the local time zone on SELECT.
What do you think? Discussion in Deutsch is ok :)

**Comment by Hofei90** [comment]
Yes, your described problem I have also always 2x a year with my weather data and smartmeter data. So far I ignored it skillfully away. But you are right, the UTC time will be the better format. But I have to check if it comes to other unexpected problems.

Changed it would be fast.
[Line 202](https://github.com/Hofei90/smartmeter/blob/master/smartmeter.py#L202) must be replaced in       
`now = datetime.datetime.utcnow()`

-----


Ja, dein beschriebenes Problem habe ich auch immer 2x im Jahr mit meinen Wetterdaten und smartmeter Daten. Bisher ignorierte ich es gekonnt weg. Aber du hast Recht, die UTC Zeit wird das bessere Format sein. Muss jedoch mal prüfen ob es dann zu anderen unerwarteten Problemen kommt.

Geändert wäre es schnell.
[Zeile 202](https://github.com/Hofei90/smartmeter/blob/master/smartmeter.py#L202) muss ersetzt werden in       
`now = datetime.datetime.utcnow()`
